prometheus-ksonnet/lib/nginx.libsonnet

```diff
@@ -11,13 +11,25 @@
             # Allow websocket connections https://www.nginx.com/blog/websocket-nginx/
             proxy_set_header    Upgrade $http_upgrade;
             proxy_set_header    Connection "Upgrade";
-    ||| else '',
+    ||| else '' + if subfilter then |||
+      sub_filter 'href="/' 'href="/%(path)s/';
+      sub_filter 'src="/' 'src="/%(path)s/';
+      sub_filter 'endpoint:"/' 'endpoint:"/%(path)s/';  # for XHRs.
```

Comment:
i dont think that a pagerbeuty specific filter should be in this ksonnet library.
Can we change the lib so that you can just pass a custom nginx config snippet instead?

Reply:
These aren't pagerbeautry specific (even if thats the reason we added them).  This should work for anything using react AFAICT.